src/containers/Workspace.js

```diff
@@ -9,6 +9,7 @@ import Workspace from '../components/Workspace';
  */
 const mapStateToProps = state => (
   {
+    isWorkspaceControlPanelVisible: state.config.workspaceControlPanel.enabled,
```

Comment:
I could theoretically pass this from `App.js` instead of getting it from state (since it's a prop there), but I wasn't sure if that would be coupling to the parent (since in order to move Workspace somewhere you may have to figure out how to get that out of state again).

I'm happy to go either way.

Reply:
State makes sense to me. I think there's still a lot of open questions about how the config is getting passed around.